srcpkgs/crun/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'crun'
+pkgname=crun
+version=0.13
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --enable-dynamic --disable-systemd"
+hostmakedepends="pkg-config libtool python3 go-md2man"
+makedepends="libcap-devel libseccomp-devel yajl-devel"
+short_desc="Lightweight OCI runtime written in C"
+maintainer="Imran Khan <imrankhan@teknik.io>"
+license="GPL-2.0-or-later, LGPL-2.1-or-later"
+homepage="https://github.com/containers/crun"
+distfiles="https://github.com/containers/crun/releases/download/${version}/crun-${version}.tar.gz"
+checksum=a7a9f458fa4c13fc63f8e74c6ce660f9c439022cd50b4a00902c258ef08e75ff
+
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	makedepends+=" argp-standalone"
+fi
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --host=${XBPS_CROSS_TRIPLET}"
```

Comment:
Isn't this should be set by `build_style=gnu-configure`?

Reply:
I see. Always wondered why there is both `configure` and `gnu-configure`. Fixed and upgraded to recent upstream version, if there is interest.